templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h3>Are you sure want to delete manufacturer {{ manufacturer.name }}?</h3>
+  <p style="color: red">All related data will be erased as well</p>
+  <form action="" method="post" >
+    {% csrf_token %}
+    <input type="submit" class="bnt btn-danger" value="Yes, I sure">
```

Comment:
Add `Cancel` button as well as mentioned in the checklist. Make sure to update the corresponding screenshot. Besides that, you could extract the part that relates to deletion operation and reuse it instead of duplication in car deletion HTML.